masq_lib/src/shared_schema.rs

```diff
@@ -15,7 +15,11 @@ pub const BLOCKCHAIN_SERVICE_HELP: &str =
      exit services from your MASQ Node (e.g. http://localhost:8545, \
      https://ropsten.infura.io/v3/YOUR-PROJECT-ID, https://mainnet.infura.io/v3/YOUR-PROJECT-ID), \
      https://base-mainnet.g.alchemy.com/v2/d66UL0lPrltmweEqVsv3opBSVI3wkL8I, \
```

Comment:
I notice that this actually contains a real project ID, where none of the others do, and @czarte notices that it's v2 instead of v3. Is any of that important?